package.json

```diff
@@ -37,7 +37,7 @@
   "homepage": "https://github.com/single-spa/import-map-deployer#readme",
   "dependencies": {
     "@azure/storage-blob": "^12.1.2",
-    "@google-cloud/storage": "^3.0.3",
+    "@google-cloud/storage": "^5.7.4",
```

Comment:
I think this upgrade is okay, despite the breaking changes listed below. I don't think any of the breaking changes affect the import-map-deployer or its users.

https://github.com/googleapis/nodejs-storage/blob/master/CHANGELOG.md#-breaking-changes
https://github.com/googleapis/nodejs-storage/blob/master/CHANGELOG.md#-breaking-changes-1

Reply:
@joeldenning nope, I have been running this and testing all features. This repos use of features, works fine